newsletter-december-2023.rst

```diff
@@ -0,0 +1,42 @@
+.. post:: Dec 6, 2022
+   :tags: newsletter, python
+   :category: Newsletter
+   :author: Eric
+
+Read the Docs newsletter - December 2023
+========================================
+
+News and updates
+----------------
+
+* We have shipped :doc:`single version projects </introducing-support-for-version-only-projects>` to allow projects to be versioned without having translations. This is a long-requested feature that we've excited to ship based on our Proxito refactor work.
+* We improved our webhook security by :doc:`requiring a secret </security-update-on-incoming-webhooks>` to be configured for all webhooks. This will help prevent malicious actors from triggering builds and other actions.
+* We fixed an `XSS vulnerability <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-9vh9-cxm2-p2c4>`_ that was reported by an external reporter: Surya Dev Singh. Thanks for the report!
+* Work continues on `Addons <https://github.com/readthedocs/addons/>`_, where we've added support for explicit EthicalAds placements on documentation pages, and are working on a new interface for configuring Addons.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+Upcoming changes
+----------------
+
+* We are working to expand the functionality of our `redirects feature <https://github.com/readthedocs/readthedocs.org/pull/10825>`_ to support more use cases. This work is close to shipping.
+* We working on the `upgrade to our dashboard notification system <https://github.com/readthedocs/readthedocs.org/pull/10890>`_, so that users have more control and better context for on-site notifications.
```

Comment:
```suggestion
* We continue working on the `upgrade to our dashboard notification system <https://github.com/readthedocs/readthedocs.org/pull/10890>`_, so that users have more control and better context for on-site notifications.
```